Some schema fixes

anderseknert opened the pull request:
These don't matter to the type checker, so wasn't caught before.

<!--
Thank you for submitting a pull request to Regal!

If you're new to contributing to the project, some tips and pointers are provided in the
contributing](https://github.com/StyraInc/regal/blob/main/docs/CONTRIBUTING.md) docs. If you find anything missing, or
not made clear enough, that's a bug, and we'd appreciate hearing about it!

If you want to ask questions before submitting your PR, or want to discuss Regal in general, please feel free to join
us in the `#regal` channel in the [Styra Community Slack](https://communityinviter.com/apps/styracommunity/signup).
-->

srenatus reviewed (approved):
How did you notice if the type checker doesn't care? 💭

anderseknert commented:
I moved the schema to the bundle directory, added this snippet to one of the entrypoints:
```rego
print(json.match_schema(input, data.regal.schema))
```
then ran `go run main.go lint --enable-print bundle` and watched the cascade of errors :) 

Should really look into making that part of the tests, but it's a little tricky as the schemas are placed where they are to appease Go's`embed` directive and its limitations on where it can embed files from... and I'd rather not have to deal with duplicates as I'd most certainly would forget to change things in two places. But I'm sure it's all solveable with some more time.
